pyproject.toml

```diff
@@ -30,8 +30,13 @@ httpx = "^0.18.2"
 hypothesis = "^6.14.0"
 black = "^21.6b0"
 watchgod = "^0.7"
-
+taskipy = "^1.9.0"
 
 [build-system]
 requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
+
+[tool.taskipy.tasks]
+start = "uvicorn api.main:app --host 0.0.0.0 --port 8000 --reload"
```

Comment:
I would like to keep `--reload` into a separate command, for example we don't need `--reload` while production.

Secondly I will let the user choose the host and port for themselves, if not given uvicorn would anyways get the default ones.

```toml
[tool.taskipy.tasks]
start = { cmd = "uvicorn main:app", help = "Run the API" }
debug = { cmd = "uvicorn main:app --reload", help = "Run the API with reloading enabled" }
```